src/components/DataView/DataViewDisplayList.tsx

```diff
@@ -30,35 +28,19 @@ interface DataViewDisplayListProps {
 	additionalActions?: any;
 	primaryActions?: any;
 	onCheckboxClick?: any;
+	transformedData?: any;
+	activeColumnObjs?: any;
 }
 
 function DataViewDisplayList(props: DataViewDisplayListProps) {
-	// todo validate props

```

Comment:
Looking at DataView.tsx I'm not completely sure why these variables were deleted from here. It seems the only thing that changes is that now that processing it's done in DataView but the values don't seem to be needed anywhere else yet so why not keep them here?